modules/node_modules/@ncigdc/containers/FileTable.js

```diff
@@ -28,7 +28,13 @@ export const FileTableComponent = (props: TTableProps) => {
 
   return (
     <div>
+
       <Row style={{ backgroundColor: 'white', padding: '1rem', justifyContent: 'space-between' }}>
+        {props.isCartPage &&
```

Comment:
Because the FileTable probably shouldn't know about the Cart. You want to add features, it's very possible that someone will want to use the Filetable with a title and the ability to add files to cart.